.pkg/installer.sh

```diff
@@ -101,8 +101,17 @@ rmfile() {
   fi
 }
 
+_rmdir() {
+  if [ -d "$1" ]; then
+    sudo rm -rf "$1"
```

Comment:
do we need sudo here?

Reply:
"normally" not for the files we currently call this function for
But in the future we could need sudo for certain files
Anyway, uninstall script need 'sudo', and at that point of script execution, sudo already has been granted by the user

Reply:
If we don't need it now, let's not add it now? Let's not try to guess what the future needs, let's add it when it's needed?

Reply:
thx for the suggestion but I prefer to keep that function "sudo" ready